packages/mendel-requirify/index.js

```diff
@@ -33,10 +28,23 @@ function requirify(b, opts) {
                 return done();
             }
 
-            var dest = path.join(outdir, row.variation, row.id);
-            var out = fs.createOutputStream(dest);
-            out.write(mendelRequireTransform(row.source, true));
-            out.end(done);
+            var match;
+            dirs.some(function(dir) {
+                var parts = file.split('/' + dir + '/');
+                var found = parts.length > 1;
+                if (found) {
+                    match = [outdir, dir, parts[1]];
+                }
+                return found;
+            });
```

Comment:
This looks very much with `variationMatches`. In case it is the same you can re-use the function.